src/ts/components/Tooltip.tsx

```diff
@@ -54,13 +54,11 @@ export const getPopperConfig = (
 };
 
 const tooltipIconView = css`
-    padding: 2px;
-    height: 20px;
+    height: 18px;
```

Comment:
I think the padding had n impact in the `i` being very slightly off centre.
This seems to have done the trick :)